packages/frontend/src/lib/ExampleCard.svelte

```diff
@@ -0,0 +1,111 @@
+<script lang="ts">
+import type { Example } from '/@shared/src/models/examples';
+import { faArrowDown, faArrowUpRightFromSquare } from '@fortawesome/free-solid-svg-icons';
+import { bootcClient } from '/@/api/client';
+import { Button } from '@podman-desktop/ui-svelte';
+import { router } from 'tinro';
+import DiskImageIcon from './DiskImageIcon.svelte';
+
+export let example: Example;
+
+let pullInProgress = false;
+
+async function openURL(): Promise<void> {
+  await bootcClient.openLink(example.repository);
+}
+
+async function pullImage(): Promise<void> {
+  if (example.image) {
+    pullInProgress = true;
+    bootcClient.pullImage(example.image);
+  }
+}
+
+async function gotoBuild(): Promise<void> {
+  if (example.image && example.tag) {
+    router.goto(`/disk-images/build/${encodeURIComponent(example.image)}/${encodeURIComponent(example.tag)}`);
+  }
+}
+
+// Function that takes something in MB and returns it in GB is more than 1GB
+// in a nice string format
+function formatStringSize(size: number): string {
+  if (size > 1000) {
+    return `${(size / 1000).toFixed(1)} GB`;
+  }
+  return `${size} MB`;
+}
```

Comment:
todo: please use humansize library

https://www.npmjs.com/package/human-size

Reply:
I decided to use the filesize library https://www.npmjs.com/package/filesize which is used in podman desktop. the one you linked hasn't been updated in 10 years.

Reply:
ah yes I think I took the wrong name but you figured it out
I mixed humanize-duration and this one